<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelInfo</class>
 <widget class="QWidget" name="ChannelInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>173</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLineEdit" name="txtTitle"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPlaying">
     <property name="text">
      <string>Now playing:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblGameStats">
     <property name="text">
      <string>Game statistics:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblNumGameChannels">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblGameChannels">
       <property name="text">
        <string>total channels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblNumGameViewers">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblGameViewers">
       <property name="text">
        <string>total viewers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="cbbPlaying">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="layChannelStats">
     <item>
      <widget class="QLabel" name="lblNumSubscribers">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSubscribers">
       <property name="text">
        <string>subscribers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblNumFollowers">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFollowers">
       <property name="text">
        <string>followers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblNumViewers">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 127)</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblViewers">
       <property name="text">
        <string>viewers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>Broadcast title:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="txtChannel">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblChannelStats">
     <property name="text">
      <string>Statistics:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblChannel">
     <property name="text">
      <string>Channel:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
